### ISSUE TITLE
Styles of Font Awesome in react-icons@3.0.1

### ISSUE THREAD

**Issue by dparisotto-starcard** [react-icons/react-icons#158]
Hi, how can i use the regular style of an icon in Font Awesome?

[regular](https://fontawesome.com/icons/heart?style=regular)
[solid](https://fontawesome.com/icons/heart?style=solid)

Thank you

**Comment by kamijin-fanta** [comment]
It is not included now.
I think it should be added.

**Comment by justswim** [comment]
I can't seem to use the regular icons  - can you please show an example of how to use the regular, outline (not solid) heart?

**Comment by pnicolli** [comment]
@justswim They are not included at this time (if you check [here](https://github.com/react-icons/react-icons/blob/master/packages/react-icons/src/icons/index.js), you can see it only exports brands and solid), but I think they should eventually be added.

Removing support for regular icons is causing issues in the migration from version 2. For example, I had `FaCircle` and `FaCircleO` in my app, but in FA5 `FaCircleO` is gone since it's just the regular version of `FaCircle`, so I can't migrate that icon at all.

**Comment by Slapbox** [comment]
@gorangajic @kamijin-fanta is there a plan to address this? Just started using `react-icons` and it's incredible, but this stopped me dead in my tracks.

**Comment by kamijin-fanta** [comment]
I think the work itself is easy.
Please tell me how to make Solid / Regular coexist. There are options to give some prefix / suffix.

- option 1
   - FaHeart
   - FaHeartReg or FaHeartR
- otpion 2
   - FaHeart
   - FaRegHeart or FaRHeart

**Comment by Slapbox** [comment]
I like option 1 better personally, but I agree with @justswim. Option 2 would be better for consistency with Font-Awesome's docs.

**Comment by moshensky** [comment]
+1 for consistency with font-awesome documentation

**Comment by kamijin-fanta** [comment]
please try `yarn add react-icons@next` .
FaRegTimesCircle etc. should be available!

**Comment by kamijin-fanta** [comment]
fix in react-icons@3.1.0

**Comment by flaska** [comment]
> fix in react-icons@3.1.0

Thank you so much! :) 👍

**Comment by drixie** [comment]
Thanks for the great work. 

Could you please add this to the documentation? Took me like 10 mins to find this lol.

**Comment by gunar** [comment]
**tl;dr**
```js
import { FaCopy, FaRegCopy } from 'react-icons/fa';

// Solid
<FaCopy />

// Regular
<FaRegCopy />
```
